### PR TITLE
Practice opencv

### PR DESCRIPTION
**Pull request by Denison904**

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -4,10 +4,12 @@
 
 #include "header.hpp"
 
+
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```
Comment (allnes) [inline modules/0_git/test/main.cpp:11]: Please create new branch and don't add old task

Comment (Denison904) [reply modules/0_git/test/main.cpp:11]: I didnt quite understand. I created new branch and made new commit. Do I need to clon code for each job?

Comment (allnes) [reply modules/0_git/test/main.cpp:11]: Master branch doesn't have yours source code. If you will create new task based on master branch then old task will not show in github differences. You can local check differences with help `git diff`.

Comment (Denison904) [reply modules/0_git/test/main.cpp:11]: I need change branch on master then create new branch and make commit. Do I understand correctly?

Comment (allnes) [reply modules/0_git/test/main.cpp:11]: You need to synchronize master branch of yours fork and master branch of `openvino_practice` and then create new branch

Comment (dkurt) [reply modules/0_git/test/main.cpp:11]: Just do
```
git checkout master
git checkout -b new_task
```
before new module - than commit the changes

Comment (Denison904) [reply modules/0_git/test/main.cpp:11]: I did. Do I need create new PR?

Comment (dkurt) [reply modules/0_git/test/main.cpp:11]: Yes, for a new branch

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/127

**Pull request closed without merging**